ddprof-lib/src/main/cpp/jfrMetadata.cpp

```diff
@@ -254,6 +254,7 @@ void JfrMetadata::initialize(const std::vector<std::string>& contextAttributes)
 
             << (type("datadog.ProfilerCounter", T_DATADOG_COUNTER, "Datadog Profiler Internal Counter")
                 << category("Datadog")
+                << field("startTime", T_LONG, "Start Time", F_TIME_TICKS)
```

Comment:
It's an instance of `jdk.jfr.Event` so the `startTime` is required

Reply:
The type is described only by its metadata, so, technically, it is not really required, as long as the parser does not make any assumptions and just follows the metadata.
Unfortunately, the built-in JDK JFR parser seems to make the assumption that events always have start time.

Since this event is emitted once per chunk, the cost of adding the field is almost nothing anyway.

Reply:
That is true (forgot that JFR is self descriptive via the metadata)